### ISSUE TITLE
new Field('title').attributes is not a function

### ISSUE THREAD

**Issue by ghoullier** [marmelab/ng-admin#212]
In [README#example-configuration](https://github.com/marmelab/ng-admin#example-configuration) there is the following configuration example:

```
 post.creationView()
        .addField(new Field('title') // the default edit field type is "string", and displays as a text input
            .attributes({ placeholder: 'the post title' }) // you can add custom attributes, too
            .validation({ required: true, minlength: 3, maxlength: 100 }) // add validation rules for fields
        )
        .addField(new Field('body').type('wysiwyg')); // overriding the type allows rich text editing for the body
```

But Field class has no method attributes()

**Comment by fzaninotto** [comment]
You're probably using ng-admin 0.4 via bower, `attributes()` is only available on master (0.5-dev).

**Comment by ghoullier** [comment]
Indeed, it works with "ng-admin": "master".

Thanks
